dipy/direction/closest_peak_direction_getter.pyx

```diff
@@ -1,16 +1,24 @@
 import numpy as np
 cimport numpy as cnp
+cimport cython
 
 from warnings import warn
 
 from dipy.direction.peaks import peak_directions, default_sphere
 from dipy.direction.pmf cimport SimplePmfGen, SHCoeffPmfGen
 from dipy.reconst.shm import order_from_ncoef, sph_harm_lookup
 from dipy.tracking.direction_getter cimport DirectionGetter
+from dipy.tracking.stopping_criterion cimport (StreamlineStatus,
+                                               StoppingCriterion,
+                                               TRACKPOINT,
+                                               ENDPOINT,
+                                               OUTSIDEIMAGE,
+                                               INVALIDPOINT,
+                                               PYERROR)
```

Comment:
I do not think this import is needed. This is not used

Reply:
yes, thx.